src/makeCalculator.js

```diff
@@ -37,7 +37,45 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    result: 0,
+
+    add(value = 0, obj = this) {
```

Comment:
you don't need to set a default value as `this` for `obj`

Reply:
fixed